src/main/java/no/digipost/monitoring/logging/LoggerThresholdMetric.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) Posten Norge AS
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.monitoring.logging;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+
+import java.time.Duration;
+
+/**
+ *
+ * `new LoggerThresholdMetric("ROOT, 1, "error", Duration.ofMinutes(5))`
+ * Example output:
+ * # HELP log_events_1min_threshold
+ * # TYPE log_events_1min_threshold gauge
+ * log_events_5min_threshold{level="error",logger="ROOT",} 1.0
+ * <p>
+ * <p>
+ * Example error alert:
+ * - alert: MyLoggingAlert
+ * expr: &gt;
+ * sum by (job,name,level,logger) (increase(logback_logger_events_total[5m]))
+ * &gt;=
+ * max by (job,name,level,logger) (log_events_5min_threshold)
+ */
+class LoggerThresholdMetric implements MeterBinder {
+
+    private final String loggerName;
+    private final double threshold;
+    private final String level;
+    private final Duration duration;
+
+    Gauge thresholdGauge;
+
+    LoggerThresholdMetric(String loggerName, double threshold, String level, Duration duration) {
```

Comment:
Package private. In the future this may be public to allow for custom duration (with the prerequisite that we want to use `log_events_<minutes>min_threshold` as naming pattern.). 
For now we want to be in control and only have metrics for 5 min duration (set via `LogbackLoggerMetrics`).